test/libsolidity/gasTests/abiv2_optimised.sol

```diff
@@ -17,9 +17,9 @@ contract C {
 // optimize-yul: true
 // ----
 // creation:
-//   codeDepositCost: 588800
-//   executionCost: 619
-//   totalCost: 589419
+//   codeDepositCost: 642000
+//   executionCost: 676
+//   totalCost: 642676
```

Comment:
One would expect this is due to specialised functions but none of the functions in the test contract are actually called,

Reply:
Some of the abi functions are getting specialized here. I can show a diff later.